code/03_census_and_bed_days.R

```diff
@@ -0,0 +1,868 @@
+#                                 if_else(age_grp=="All" & 
+#                                           reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                                                    "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,delay_under_2_weeks)))
+# 
+# datafile7<-datafile7 %>% mutate(acute,
+#                                 if_else(age_grp=="All" & 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,acute)))
+#                                                                       
+# datafile7<-datafile7 %>% mutate(gpled,
+#                                 if_else(age_grp=="All" & 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,gpled)))
+#                                                                       
+#                                                                       
+#                                                                                                                                          
+# 
+# datafile7<-datafile7 %>% mutate(notgpled,
+#                                 if_else(age_grp=="All" & 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+# 
+# datafile7<-datafile7 %>% mutate(acute,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,acute)))
+# 
+# datafile7<-datafile7 %>% mutate(gpled,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+# 
+# 
+# datafile7<-datafile7 %>% mutate(notgpled,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+
+#Note: used datafile6 as the creation of datafile7 doesn't add anything new - PMc: check .
+
+### 13. Save out census file ----
+
+write_sav(datafile6,paste0(filepath,"All census data.sav"))
+
+write.xlsx(datafile6,paste0(filepath,"All census data.xlsx"))
```

Comment:
See "Style Guide" section in general feedback.
```suggestion
write.xlsx(datafile6, here::here("outputs", "All census data.xlsx"))
```

Reply:
I'd just write this out as a csv using `readr::write_csv` to keep things simple. Also, I don't think an outputs folder is created in the setup-environment script. I'd suggest we add a bit of code to create this and a dated folder within.